1.10/networking/load-balancing-vips/index.md

```diff
@@ -1,134 +1,50 @@
 ---
-post_title: Load Balancing and VIPs
-nav_title: Load Balancing and VIPs
+post_title: Load Balancing and Virtual IPs (VIPs)
 menu_order: 00
 ---
 
-DC/OS comes with an east-west load balancer that's meant to be used to enable multi-tier microservices architectures. It acts as a TCP Layer 4 load balancer, and it's tightly integrated with the kernel. 
+DC/OS provides an east-west load balancer (minuteman) that enables multi-tier microservices architectures. It acts as a TCP layer 4 load balancer, and it's tightly integrated with the Mesos kernel. The features include:
```

Comment:
s/tightly integrated with Mesos kernel/ leverage load-balancing features within the linux kernel to achieve near line-rate throughputs and latency.